core/arch/arm/include/arm.h

```diff
@@ -76,6 +76,9 @@
 #define MPIDR_AARCH32_AFF_MASK	(MPIDR_AFF0_MASK | MPIDR_AFF1_MASK | \
 				 MPIDR_AFF2_MASK)
 
+/* MPIDR definitions for VCPU */
+#define MPIDR_VCPU_MASK     ULL(0xffffff)
```

Comment:
Could you change the indentation from space chars to 2 tabulations?

Reply:
Fixed to 2 tabulations